streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/executiongraph/ExecutionJobVertex.java

```diff
@@ -137,21 +165,13 @@ public boolean isSinkVertex() {
     return getVertexType() == VertexType.SINK;
   }
 
-  public JobRuntimeContext getRuntimeContext() {
-    return runtimeContext;
-  }
-
   @Override
   public String toString() {
     return MoreObjects.toStringHelper(this)
-        .add("jobVertexId", jobVertexId)
-        .add("jobVertexName", jobVertexName)
-        .add("streamOperator", streamOperator)
+        .add("jobVertexId", operatorId)
```

Comment:
If we replace all jobVertexId by operatorId, will it be better we don't use jobVertexId any more? Otherwise it would be confusing and user may want to know the difference between jobVertexId/operatorId

Reply:
LGTM.